src/app/api/file/route.ts

```diff
@@ -0,0 +1,362 @@
+import { NextResponse } from "next/server";
+import { File, FileResponse } from "./route.schema";
+import { google } from "googleapis";
+import { prisma } from "@server/db/client";
+import { withSession } from "../../../server/decorator";
+import { env } from "../../../env/server.mjs";
+
+const initializeDriveAuth = async (request) => {
```

Comment:
Since googleapis are shared between multiple files, should we move this to a different file? One potential option is `@server/service`. What do you guys think?

Reply:
We agree - a separate file is a good idea, especially because senior functionality will do the same thing. We fixed the double duty as well!! Thank you Nick!